app/App.hs

```diff
@@ -36,18 +38,21 @@ main = do
     -- TODO: Use a logging interface instead of using putStrLn.
     putStrLn "Started..."
 
-    let confPolicyMap = Map.fromList [(CM.script_hash x, x) | x <- scripts]
-        conn = mkLocalNodeConnectInfo networkId socketPath
-    streamChainSyncEvents conn points
-        & Stream.filter (not . isByron)
-        & fmap getEventTransactions
-        & Stream.postscanl trackPreviousChainPoint
-        -- TODO: Try to replace "concatMap" with "unfoldEach".
-        & Stream.concatMap (Stream.fromList . (\(a, b) -> (a,) <$> b))
-        & Stream.mapM (mkContext1 conn . uncurry mkContext0)
-        & Stream.filter
-            ( \ctx1@Context1{..} ->
-                not . Map.null . Map.restrictKeys confPolicyMap $
-                    Set.union (getMintPolicies context0) (getSpendPolicies ctx1)
-            )
-        & Stream.fold (Fold.drainMapM print)
+    withAsync (HTTP.run httpServerPort) $ \serverAsync -> do
+      link serverAsync
```

Comment:
You can do this with streamly as well:

```
Stream.parList (stopWhen FirstStops)
    [ Stream.fromEffect (Http.run port)
    , Stream.fromEffect streaming
    ]
```

Feel free to use whatever looks cleaner to you.